CONTRIBUTING.md

```diff
@@ -1,22 +1,37 @@
 # Contribution guide
 
-## How to start?
-
-Contributing is quite easy: suggest ideas and make them done.
-We use [GitHub issues](https://github.com/etna-team/etna/issues) for bug reports and feature requests.
+Let's look at the possible ways to contribute to the project:
+- Voting for suggestions
+  - Vote for the suggestion you are interested in on [GitHub Discussions](https://github.com/etna-team/etna/discussions/categories/improvements)
+- Taking part in discussions
+  - Take part in discussions on [GitHub Discussions](https://github.com/etna-team/etna/discussions)
+- Improving documentation
+  - Create an [issue about improving documentation](https://github.com/etna-team/etna/issues/new/choose)
+- Sending bug report
+  - Create an [issue about bug](https://github.com/etna-team/etna/issues/new/choose)
+- Suggesting an idea
+  - If there is suggestion with very similar idea on [GitHub Discussions](https://github.com/etna-team/etna/discussions/categories/improvements), vote for it
+  - Otherwise, add new idea to [GitHub Discussions](https://github.com/etna-team/etna/discussions/categories/improvements)
+  - You could continue this step by taking part in discussion
+- Pointing out a problem
+  - If there is suggestion with very similar problem on [GitHub Discussions](https://github.com/etna-team/etna/discussions/categories/improvements), vote for it
+  - Otherwise, add new problem to [GitHub Discussions](https://github.com/etna-team/etna/discussions/categories/improvements)
+  - You could continue this step by taking part in discussion
+- Making a pull request
```

Comment:
1. May be we should place it on top as PR is the most valuable contribution
2. We should highlight when this option is relevant(i.e. you want to implement already specified issue)

Reply:
- Valid point. But I thought it may be less scary to order them by difficulty from easy to hard.
- Ok, I'll do it.